react-form-controls/form-controls-inside-of-react/src/RegistrationFormUncontrolled.tsx

```diff
@@ -0,0 +1,56 @@
+// import files
+import { FormEvent } from 'react';
+
+export function RegistrationFormUncontrolled() {
+  const bl = {
+    display: 'inline-flex',
+    flexDirection: 'column',
+    margin: '15px',
+  };
+  const blA = {
+    display: 'block',
+    margin: '10px auto',
+  };
+
+  function handleSubmit(e): FormEvent<HTMLFormElement> {
+    e.preventDefault();
+
+    const formReciever = document.getElementById('my_form');
```

Comment:
In React, never access the `document` object directly. You can get the form using `e`. (e.currentTarget)